index.js

```diff
@@ -1,10 +1,11 @@
 import { NativeModules, NativeEventEmitter } from 'react-native';
-import { version } from './package.json';
+import packageJson from './package.json';
 import { isContext, validateContext } from './src/contextUtils';
 import { LDInvalidUserError } from 'launchdarkly-js-sdk-common/src/errors';
 import { invalidContext } from 'launchdarkly-js-sdk-common/src/messages';
 
 let LaunchdarklyReactNativeClient = NativeModules.LaunchdarklyReactNativeClient;
+const { version } = packageJson
```

Comment:
Hello @theParthe, thank you for the contribution.

Because the code is trying to get the version (instead of the whole package.json's content), shouldn't this line say packageJson.version?